resources/user-guides/attach-data.md

```diff
@@ -251,24 +256,24 @@ After you complete your configuration, verify that your rule looks like the foll
 1. Click **Save**
 2. Rebuild your Launch property and deploy it to the correct Environment.
 
-### Attaching additional Target parameters to Target location-clicked events
+### Attaching additional data to Target location-clicked events<a name="Target-Attach-Data-Location-Clicked"></a>
```

Comment:
Please update the heading to:
"Attach additional data to Target location-clicked events"